DNS for SVCB-reliant proxy usage (#295)
MikeBishop/dns-alt-svc: draft-ietf-dnsop-svcb-https.md

(e.g. through an affiliated DNS resolver), the client would have to perform

> SVCB queries though a separate resolver.  This might disclose the client's
> destinations to an additional party, creating privacy concerns.  If these

Reviewer: ```suggestion
SVCB resolution separately, likely disclosing the destinations to additional parties.
```

> concerns apply, SVCB-optional clients SHOULD disable SVCB resolution
> unless they arrange for SVCB resolution with appropriate privacy 
> and routing properties.  As SVCB-reliant clients cannot function in this 
> configuration, they must arrange for SVCB resolution with appropriate 
> privacy and routing properties.

Reviewer: ```suggestion
Clients that support such proxies SHOULD arrange for a separate SVCB resolution
procedure with appropriate privacy properties, or disable SVCB resolution entirely if
SVCB-optional.
```

(e.g. through an affiliated DNS resolver), the client would have to perform
SVCB queries though a separate resolver.  This might disclose the client's

> destinations to an additional party, creating privacy concerns.  If these

Author: Need to cleanup.
```suggestion
destinations to an additional party, creating privacy concerns.
```

> concerns apply, the client SHOULD mitigate these concerns in some way,

Reviewer: ```suggestion
concerns apply, SVCB-optional clients SHOULD disable SVCB resolution.
SVCB-reliant clients cannot function in this configuration, so they must
arrange for SVCB resolution with appropriate privacy and routing properties.
```

Author: Perhaps we also allow for "SVCB-optional clients SHOULD disable SVCB resolution
unless they arrange for SVCB resolution with appropriate privacy and routing properties."
